src/routes/overviews/bufnmf.svx

```diff
@@ -0,0 +1,134 @@
+---
+title: Audio Decomposition using BufNMF
+blurb: |
+    Decompose an audio buffer into component parts using non-negative matrix factorization.
+tags:
+    - decomposition
+flair: article
+---
+
+<script>
+    import Image from '$lib/components/Image.svelte';
+    import Audio from '$lib/components/Audio.svelte';
+</script>
+
+# Decomposition
+
+"NMF" stands for Non-negative Matrix Factorization, which is a machine learning algorithm that can be used for decomposing an audio buffer into component parts. In this example, we’ll look at decomposing a mono drum loop into three different mono audio buffers, each hopefully containing the hits from just one of the instruments: kick, snare, and cymbal. Here is the recording and spectrogram of our drum loop.
+
+<Audio
+src="/audio/Nicol-LoopE-M.mp3"
+label="Mono drum loop included in the FluCoMa package."
+/>
+
+<Image
+src="/overviews/bufnmf/00_original_spectrogram_mel_color.png"
+label="Mel-frequecy spectrogram of drum loop"
+/>
+
+The spectrogram below roughly identifies where in time and frequency each of these instruments are present.
+
+<Image
+src="/overviews/bufnmf/00_boxes_over_nicol-01.png"
+label="(red = kick, blue = snare, yellow = cymbal)"
+/>
+
+As you can see (and hear in the playback), these different drum hits overlap quite a lot in both time and spectrum meaning that a purely time-based (such as a slicer) or purely spectral (such as a filter) strategy for separating these hits wouldn’t work very well.
+
+This is where Non-negative Matrix Factorization can be quite useful.
```

Comment:
This should be all lower (non-negative matrix factorisation) as well as british english.